app/models/ext_management_system.rb

```diff
@@ -317,6 +318,7 @@ def validate_zone_not_maintenance_when_ems_enabled?
   supports_attribute :feature => :storage_services
   supports_attribute :feature => :add_storage
   supports_attribute :feature => :add_host_initiator
+  supports_attribute :feature => :add_host_initiator_group
```

Comment:
E.g. this could be `supports_attribute :supports_create_host_initiator_group, :child_model => "HostInitiatorGroup"`
Then you would have `supports :create` in your provider's `HostInitiatorGroup` subclass

Reply:
Thanks for the easy-to-follow instructions!